public/static/docs/command-reference/import.md

```diff
@@ -158,23 +158,8 @@ deps:
 
 If the
 [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
-moves (e.g. a branch), you may use `dvc update` to bring the data up to date.
-However, for typically static references (e.g. tags), or for SHA commits, in
-order to actually "update" an import, it's necessary to **re-import the data**
-instead, by using `dvc import` again without or with a different `--rev`. This
-will overwrite the import stage (DVC-file), either removing or replacing the
-`rev` field, respectively. This can produce an import stage that is able to be
-updated normally with `dvc update` going forward. For example:
-
-```dvc
-$ dvc import --rev master \
-             git@github.com:iterative/dataset-registry.git \
-             use-cases/cats-dogs
-```
-
-> In the above example, the value for `rev` in the new import stage will be
-> `master`, which happens to be the default branch in this Git repository, so
-> the command is equivalent to not using `--rev` at all.
+moves (e.g. a branch), you may use `dvc update` preferably with `--rev` option
+to bring the data up to date. Using `--rev` also updates the fixed-revision.
```

Comment:
```suggestion
to bring the data up to date.
```

Instead of that last sentence, I would keep some of the deleted text from above and change it a little bit. Somehing like:

For typically static references (e.g. tags), or for SHA commits, in order to update an import, it's necessary to use `dvc update --rev` with a different reference, as shown below. This will replace the `rev` field in the DVC-file (import stage).

```dvc
$ dvc update --rev <new-ref> \
             git@github.com:iterative/dataset-registry.git \
             use-cases/cats-dogs
```

Reply:
AND on top of that, we probably need a separate paragraph still to explain the re-import options, which is still available. Not sure cc @shcheklein (otherwise, let's remove the **re-import** term everywhere e.g. in the title of this section). Something like:

It's also possible to **re-import the data** instead, by using `dvc import` again without or with a different `--rev`. This
will overwrite the import stage (DVC-file), either removing or replacing the `rev` field, respectively. This can produce an import stage that is able to be updated normally with `dvc update` going forward. For example:

```dvc
$ dvc import --rev master \
             git@github.com:iterative/dataset-registry.git \
             use-cases/cats-dogs
```

Reply:
Yes, I agree that it should be consistent and title should be updated. It's not important to have "re-import" now. It def worth mentioning that it overwrites the file if you run it again, though. Not sure about the example section - let you decide :)

Reply:
I vote for removing the "re-import" term everywhere then (including from the corresponding link hrefs).